Formula/mumps.rb

```diff
@@ -0,0 +1,123 @@
+class Mumps < Formula
+  desc "Parallel Sparse Direct Solver"
+  homepage "http://mumps-solver.org"
+  url "http://mumps.enseeiht.fr/MUMPS_5.1.2.tar.gz"
+  sha256 "eb345cda145da9aea01b851d17e54e7eef08e16bfa148100ac1f7f046cd42ae9"
+
+  depends_on "open-mpi"
+  depends_on "openblas"
+  depends_on "gcc"
+  depends_on "metis"
+
+  resource "mumps_simple" do
+    url "https://github.com/dpo/mumps_simple/archive/v0.4.tar.gz"
+    sha256 "87d1fc87eb04cfa1cba0ca0a18f051b348a93b0b2c2e97279b23994664ee437e"
+  end
+
+  def install
+    make_args = ["RANLIB=echo"]
+    # Building dylibs with mpif90 causes segfaults on 10.8 and 10.10. Use gfortran.
+    shlibs_args = ["LIBEXT=.dylib",
+                   "AR=gfortran -dynamiclib -Wl,-install_name -Wl,#{lib}/$(notdir $@) -undefined dynamic_lookup -o "]
+    make_args += ["OPTF=-O", "CDEFS=-DAdd_"]
+    orderingsf = "-Dpord"
+
+    makefile = "Makefile.G95.SEQ"
+    cp "Make.inc/" + makefile, "Makefile.inc"
+
+    make_args += ["SCOTCHDIR=#{Formula["scotch"].opt_prefix}",
+                  "ISCOTCH=-I#{Formula["scotch"].opt_include}"]
+
+    scotch_libs = "LSCOTCH=-L$(SCOTCHDIR) -lscotch -lscotcherr -lscotcherrexit"
+    scotch_libs += "-lscotchmetis"
+    make_args << scotch_libs
+    orderingsf << " -Dptscotch"
+
+    make_args += ["LMETISDIR=#{Formula["metis"].opt_lib}",
+                  "IMETIS=#{Formula["metis"].opt_include}",
+                  "LMETIS=-L#{Formula["metis"].opt_lib} -lmetis"]
+    orderingsf << " -Dmetis"
+
+    make_args << "ORDERINGSF=#{orderingsf}"
+
+    make_args += ["CC=#{ENV["CC"]} -fPIC",
```

Comment:
`-fPIC` is not needed here, as far as I can tell. It was useful on Linux, but not on macOS